I/isoband/build_tarballs.jl

```diff
@@ -31,4 +31,4 @@ dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
Ok, this doesn't make much sense to me.  I'd try using GCC 5 and see what happens
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"5")
```

Reply:
Yeah, using GCC 5 the library doesn't expose any string at all.  It doesn't make any sense at all.